build/pxc-configure-pxc.sh

```diff
@@ -69,6 +69,10 @@ MYSQL_VERSION=$(mysqld -V | awk '{print $3}' | awk -F'.' '{print $1"."$2}')
 if [ "$MYSQL_VERSION" == '8.0' ]; then
 	grep -E -q "^[#]?admin-address" "$CFG" || sed '/^\[mysqld\]/a admin-address=\n' ${CFG} 1<>${CFG}
 	grep -E -q "^[#]?log_error_suppression_list" "$CFG" || sed '/^\[mysqld\]/a log_error_suppression_list="MY-010055"\n' ${CFG} 1<>${CFG}
+    if [ "$HIGH_COMPRESSION" == 'yes' ]; then
+        sed -r "s|^[#]?default_storage_engine=.*$|default_storage_engine=BLACKHOLE|" ${CFG} 1<>${CFG}
```

Comment:
This is awesome! Very neat solution.
I assume it reduces the time to backup and restore to almost 0.